Objects/perf_trampoline.c

```diff
@@ -149,6 +149,10 @@ typedef enum {
 #include <sys/types.h>
 #include <unistd.h>
 
+#if defined(__arm__) || defined(__arm64__) || defined(__aarch64__)
```

Comment:
Maybe only define the function if it does something? Move ``#if defined(__clang__) || defined(__GNUC__)`` here?